sopel/coretasks.py

```diff
@@ -1071,6 +1187,7 @@ def end_who(bot, trigger):
 @module.thread(False)
 @module.unblockable
 def track_notify(bot, trigger):
+    """Track user going away."""
```

Comment:
```suggestion
    """Track users going away or coming back."""
```

At least I _assume_ that `AWAY` events are sent also for coming back, based on the actual code.